opencti-platform/opencti-graphql/src/utils/format.js

```diff
@@ -122,6 +122,14 @@ export const truncate = (str, limit = DEFAULT_TRUNCATE_LIMIT, withPoints = true)
   return `${trimmedStr.substr(0, Math.min(trimmedStr.length, trimmedStr.lastIndexOf(' ')))}...`;
 };
 
+const formatSoftware = (stixCyberObservable) => {
+  const value = stixCyberObservable.name || stixCyberObservable.cpe || stixCyberObservable.swid || 'Unknown';
```

Comment:
I think you should get the representative ?

Reply:
isn't this where representations are generated?

Reply:
Oh god, I switched from Celine review and I thought you were in the front aswell. Sorry :D

Reply:
is it good for you so ? :D